lib/travis/api/v3/queries/build.rb

```diff
@@ -39,7 +39,7 @@ def restart(user)
     def priority(user)
       raise NotFound, "Jobs are not found" if find.jobs.blank?
       find.jobs.update_all(priority: PRIORITY[:high])
-      return if find.owner_type == "User"
+      return if find.owner_type != "Organization"
       if cancel_all == "true"
```

Comment:
If we pass boolean then wouldn't it be better if we just do as follow

```
if cancel_all
  # perform the action
end
```

Reply:
We will be getting string values for cancel_all param.